build-config.ts

```diff
@@ -9,6 +9,8 @@ require("dotenv").config();
 
 export default {
   inputDir: process.env.COMMERCE_SDK_INPUT_DIR || `${__dirname}/apis`,
+  apiBackupDir: process.env.COMMERCE_SDK_API_BACKUP_DIR || `${__dirname}/apiBackup`,
+  diffFile: process.env.COMMERCE_SDK_RAML_DIFF_FILE || `${__dirname}/ramlDiff.json`,
```

Comment:
Maybe specify a `tmpDir` instead, and put both of these as static paths inside there. This would also eliminate the need for additional config options in potential future changes.

Reply:
We are gonna be using this every release, I feel a predictable location for this file would be convenient. I don't see any major advantage of putting the file in a tmp dir. I am going to skip this, if you don't mind.

Reply:
We are gonna be using this every release, I feel a predictable location for this file would be convenient. I don't see any major advantage of putting the file in a tmp dir. I am going to skip this, if you don't mind.

Reply:
This makes sense. Will make the change.